exercises/practice/high-scores/high_scores.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <array>
+#include <vector>
+
+using namespace std;
+
+namespace arcade {
+
+class HighScores {
+   private:
+    vector<int> scores;
+
+   public:
+    HighScores(vector<int> scores) : scores(scores){};
```

Comment:
should the implementation of the constructor be given in the initial setup?

Reply:
The original problem spec suggested putting the exercise as "easy" so I made it easier by offering the complete header. Do you think that the "burden" of handling the header file is okay for an "easy" exercise?

Reply:
I'm fine with the definition of the class given to the students. I was just wondering about the definition of the constructor because we don't do that in any other practice exercise. I'd just *declare* the constructor in the `.h` file and leave its implementation to the students. But in the end I'm OK with either way.